locations/spiders/maif_fr.py

```
@@ -0,0 +1,10 @@
+from scrapy.spiders import SitemapSpider
+
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class MaifFRSpider(SitemapSpider, StructuredDataSpider):
+    name = "maif_fr"
+    item_attributes = {"brand": "MAIF", "brand_wikidata": "Q3331029"}
+    sitemap_urls = ["https://agence.maif.fr/sitemap_pois.xml"]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)